dquartic/utils/data_loader.py

```diff
@@ -38,6 +38,7 @@ def __init__(self, parquet_directory=None, ms2_file=None, ms1_file=None, normali
             print(f"Info: Loaded  {len(self.ms2_data)} MS2 slice samples and {len(self.ms1_data)} MS1 slice samples from NPY files.")
         elif parquet_directory is not None and ms1_file is None and ms2_file is None:
             self.meta_df = self.read_parquet_meta(parquet_directory)
+            self.parquet_directory = parquet_directory  # Store the directory path
```

Comment:
Can probably remove the comment, since it's clearly code and self-explanatory.

```suggestion
            self.parquet_directory = parquet_directory  
```